core/src/main/java/pl/agh/edu/screen/ScenariosScreen.java

```diff
@@ -0,0 +1,339 @@
+				difficultyButtonFont = "white-h3";
+				backFont = "white-h1";
+				scenarioTitleFont = "white-h2";
+				diffHeight = height / 12;
+				diffWidth = width / 8;
+				largePaddingMultiplier = height / 1000f * 0.85f;
+				System.out.println(largePaddingMultiplier);
+				System.out.println(3);
+			}
+		}
+		titleLabel = createTitleLabel();
+	}
+
+	public void createScenariosFrame() {
+		scenariosTable.top();
+		setFillParent(scenariosTable);
+
+		Label label1 = new Label("Choose scenario", titleLabel);
+
+		Table titleTable = new Table();
+		titleTable.setBackground(buttonBackground);
+		titleTable.add(label1);
+		titleTable.pad(label1.getHeight() / 4, label1.getWidth() / 12, label1.getHeight() / 4, label1.getWidth() / 12);
+
+		scenariosTable.add(titleTable).center().padTop((int) (largePaddingMultiplier * height / 10)).padBottom((int) (height / 26)).expandX();
+
+		scenariosTable.row();
+
+		Table buttons = new Table();
+		buttons.add(addScenariosButton("Seaside Resort", "water", "Most of the clients are people who are on vacation", "Summer", HotelType.RESORT, scenarioTitleFont,
+				difficultyButtonFont));
+		buttons.add(addScenariosButton("Sanatorium Ciechocinek", "hospital", "Clients are mostly patients but not only", "Autumn", HotelType.SANATORIUM, scenarioTitleFont,
+				difficultyButtonFont)).padLeft((int) (width / 12));
+		buttons.add(addScenariosButton("City center Hotel", "hotel", "Hotel guests are mostly businessman", "Spring", HotelType.HOTEL, scenarioTitleFont, difficultyButtonFont))
+				.padLeft((int) (width / 12));
+		scenariosTable.add(buttons).padTop((int) (largePaddingMultiplier * height / 24));
+
+		scenariosTable.row();
+
+		TextButton.TextButtonStyle playStyle = new TextButton.TextButtonStyle();
+		playStyle.up = buttonBackground;
+		playStyle.down = onSelect;
+		playStyle.font = skin.getFont("white-h1"); // Set the custom font
+		playStyle.fontColor = Color.YELLOW; // Set the font color
+
+		Button next = new TextButton("Next", playStyle);
+		next.pad(15);
+		next.addListener(new ClickListener() {
+			@Override
+			public void clicked(InputEvent event, float x, float y) {
+				if (selected != null && currentlySelected != null) {
+					difficultyTable.removeActor(errorLabel);
+				}
+				mainTable.clearChildren();
+				mainTable.add(difficultyTable).growX();
+			}
+		});
+
+		scenariosTable.add(next).right().padRight((int) (width / 12)).padTop((int) (largePaddingMultiplier * height / 16));
+
+		// scenariosTable.debug();
+	}
+
+	private void setFillParent(Table table) {
+		table.setFillParent(true);
+	}
+
+	public ScenarioButton addScenariosButton(String title, String image, String description, String season, HotelType hotelType, String scenarioTitleFont,
+			String scenarioTextFont) {
+		ScenarioButton scenario = new ScenarioButton(title, image, description, season, hotelType, scenarioTitleFont, scenarioTextFont);
+		scenario.setTouchable(Touchable.enabled);
+		scenario.addListener(new ClickListener() {
```

Comment:
jeżeli już stworzyliśmy specjalnie klasę scenariobutton to czemu nie mieć tego wszystkiego tam?